packages/jest-mock/src/testAssets/symbolsAreCool.js

```diff
@@ -0,0 +1 @@
+module.exports = Symbol.for('Bowties.are.cool');
```

Comment:
can you move this file into `__mocks__`? That will make sure we aren't shipping this file to npm.